Client/Frontend/Extensions/DevicePickerViewController.swift

```diff
@@ -221,7 +221,7 @@ class DevicePickerViewController: UITableViewController {
     }
 
     @objc func refresh() {
-        RustFirefoxAccounts.shared.accountManager.deviceConstellation()?.refreshState()
+        RustFirefoxAccounts.shared.accountManager.peek()?.deviceConstellation()?.refreshState()
```

Comment:
Wherever possible, use `peek()` to see if the account manager exists and is setup, because that is simpler than `.uponQueue() { ... }`. If it is nil, do nothing, and that is safe to do.